### PR TITLE
M1 changes

### PR DESCRIPTION
**Pull request by mgralinski-bright**

### CODE REVIEW

File: blog/introduction.md [hunk 1]
```diff
@@ -165,18 +165,24 @@ component main = Task();
...
 Than she can generate R1CS by running the command:
 
-`circom task.circom --r1cs --wasm --sym --c --o build --O0 --p bls12381`
+```
+mkdir build
+circom task.circom --r1cs --wasm --sym -o build --O0 -p bls12381
+```
 
-This will generate a file (*task.r1cs*) which describes a R1CS in the *Cricom*. After that Alice will need to create a witness (vector $s$) file, but this time she will use a *SnarkJS* tool for doing this. First she needs to create a json file which will describe all private inputs in our circuits. In our case this is a very simple task, because our only input value is $x$ which is “*3*”. Input file (*input.json*) will look like this:
+This will generate a file (*build/task.r1cs*), which describes a R1CS in the *Cricom* and (*build/task.wasm*), which compiles circuit to WebAssembly. After that Alice will need to create a witness (vector $s$) file, but this time she will use a *SnarkJS* tool for doing this. First she needs to create a json file which will describe all private inputs in our circuits. In our case this is a very simple task, because our only input value is $x$ which is “*3*”. Input file (*input.json*) will look like this:
```
Comment (kziemianek) [inline blog/introduction.md:173]: After reading first sentence you can come to conclusion that file `task.wasm` compiles circuit to WebAssembly. Is this true?

File: pallets/zk-snarks/sample/vk.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (kziemianek) [inline pallets/zk-snarks/sample/vk.json:1]: What's the purpose of this file? It's not valid verification key. Same thing relates to proof file added in this pull request.

Comment (mgralinski-bright) [reply pallets/zk-snarks/sample/vk.json:1]: You are right, I will remove commit "Improving documentation" from this PR. I will need to create a separate PR and prepare a documentation for this.

File: blog/introduction.md (hunk 1 above)
Comment (mgralinski-bright) [reply blog/introduction.md]: You are right, please check the changes

File: blog/introduction.md [hunk 3]
```diff
@@ -165,18 +165,24 @@ component main = Task();
...
-This will generate a file (*task.r1cs*) which describes a R1CS in the *Cricom*. After that Alice will need to create a witness (vector $s$) file, but this time she will use a *SnarkJS* tool for doing this. First she needs to create a json file which will describe all private inputs in our circuits. In our case this is a very simple task, because our only input value is $x$ which is “*3*”. Input file (*input.json*) will look like this:
+This will generate a file (*build/task.r1cs*), which describes an R1CS in the *Cricom*. Another artifact is a *build/task.wasm*, which is a circuit compiled to the WebAssembly. After that Alice will need to create a witness (vector $s$) file, but this time she will use a *SnarkJS* tool for doing this. First she needs to create a json file which will describe all private inputs in our circuits. In our case this is a very simple task, because our only input value is $x$ which is “*3*”. Input file (*input.json*) will look like this:
 
 ```
 {"x": "3"}
 ```
 
-Than she can generate a witness:
-`node generate_witness.js task.wasm ../../input.json witness.wtns`
+Than she can generate a witness file (*build/witness.wtns*):
```
Comment (kziemianek) [inline blog/introduction.md:179]: I think Than -> Then

**Review by kziemianek — APPROVED** [review]

**Pull request merged**